luxonis_train/models/predefined_models/ddrnet_segmentation_model.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass, field
+
+from luxonis_train.utils.config import (
+    LossModuleConfig,
+    ModelNodeConfig,
+)
+from luxonis_train.utils.types import Kwargs
+
+from .segmentation_model import SegmentationModel
+
+
+@dataclass
+class DDRNetSegmentationModel(SegmentationModel):
+    backbone: str = "DDRNet"
+    n_classes: int = 1
+    highres_planes: int = 64
+    layer5_bottleneck_expansion: int = 2
+    aux_head_params: Kwargs = field(default_factory=dict)
+
+    @property
+    def nodes(self) -> list[ModelNodeConfig]:
+        self.backbone_params.update({"highres_planes": self.highres_planes})
+        self.backbone_params.update(
+            {"layer5_bottleneck_expansion": self.layer5_bottleneck_expansion}
+        )
+
+        self.head_params.update(
```

Comment:
When using the suggested `BaseNode` properties, the computation of `in_planes` is no longer necessary